docs/channels/library_auth_reference/pusher-websockets-protocol.md

```diff
@@ -165,7 +167,7 @@ When the client has connected to the Channel service a `pusher:connection_establ
 { "event": "pusher:connection_established", "data": String }
 ```
 
-Where the `data` field is a JSON-encoded hash of following format:
+Where the `data` field is a JSON-encoded string of following format:
```

Comment:
Why does it say `JSON-encoded hash`? I believe it should be `JSON-encoded string`. This applied to many other places. This is just an example.